app/javascript/components/patient/Patient.js

```diff
@@ -238,10 +240,23 @@ class Patient extends React.Component {
               {this.renderEditLink('Contact Information', 2)}
             </div>
             <div className="item-group">
+              {this.state.isMinor && (
+                <React.Fragment>
+                  <span className="text-danger">Monitoree is a minor{!this.props.details.head_of_household && this.props.hoh && '. '}</span>
+                  {!this.props.details.head_of_household && this.props.hoh && (
+                    <span>
+                      View contact info for head of household:
+                      <a className="pl-1" href={`${window.BASE_PATH}/patients/${this.props.hoh.id}`}>
+                        {formatName(this.props.hoh)}
+                      </a>
+                    </span>
```

Comment:
Was there a decision from PO team somewhere along the way to present the HoH this way? The screenshots in the PR description are different and look more like the mockup I found.

Current:
![image](https://user-images.githubusercontent.com/8235974/122450420-fd6f0700-cf74-11eb-8be9-2ddba8fccbb5.png)


From mockup:
![image](https://user-images.githubusercontent.com/8235974/122450015-88033680-cf74-11eb-8600-d8521bb44789.png)

In my non-expert opinion, I think the mockup looks better/is easier to parse visually, but just wanted to check if I missed a decision somewhere.

Reply:
You did: it was buried in the replies of a UAT thread. there's been a lot of going back and forth over what exactly it should look like that's been the main reason this has been in review for so long, and this is what we all eventually arrived at.